### DISCUSSION TITLE
Alternative of Colors.Accent in .NET8 Maui

### DISCUSSION THREAD

**Discussion by IXChinmayD** [dotnet/maui#21549] [Q&A]
What is the alternative for Colors.Accent from Xamarin Forms in MAUI .NET8?

**Comment by Omkar-S-Pawar** [comment]
you can use `Color.FromArgb` method to define colors in MAUI (.NET 8) using alpha, red, green, and blue components.
Here's how you can define an accent color using `Color.FromArgb`:
```csharp
var accentColor = Color.FromArgb(255, 0, 120, 212); // Alpha (255 for fully opaque), Red, Green, Blue
```
Replace `0`, `120`, and `212` with the RGB values of your desired accent color.
